### PR TITLE
Taylor signup page and index page

### PR DESCRIPTION
**Pull request by AmazingOrange**
Index page and sign up page are completed. Both pages are working on two media screen sizes: desktop and mobile.

### CODE REVIEW

File: app/html/xcreate_user.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```
Comment (mksample) [inline app/html/xcreate_user.html:1]: what's the different between create_user.html and xcreate_user.html?

File: 2800_app.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const { JSDOM } = require('jsdom');
 const { connected } = require("process");
 
 const sqlAuthentication = { // sql connection settings
-    host: "localhost",
+    host: "127.0.0.1",// for Mac os, Taylor changes this.
```
Comment (mksample) [inline 2800_app.js:10]: This should be left as localhost for now. You can keep those changes locally though

**Review by mksample — COMMENTED** [review]
Looking good to me, only left a question and a minor change request

File: app/html/xcreate_user.html (hunk 1 above)
Comment (AmazingOrange) [reply app/html/xcreate_user.html:1]: oh I should have deleted xcreate_user.html

**Review by mksample — APPROVED** [review]
LGTM!

**Pull request merged**